lib/rules/consistent-return.js

```diff
@@ -33,6 +33,18 @@ function isUnreachable(segment) {
     return !segment.reachable;
 }
 
+/**
+* Checks whether a given node is a `constructor` method in an ES6 class
+* @param {ASTNode} node A node to check
+* @returns {boolean} `true` if the node is a `constructor` method
+*/
+function isClassConstructor(node) {
+    return node.type === "FunctionExpression" &&
+        node.parent &&
+        node.parent.type === "MethodDefinition" &&
+        node.parent.kind === "constructor";
+}
```

Comment:
I was searching for what other rules use `astUtils.isES5Constructor` (to see if those uses would also apply to checking for ES6 constructors), but I found that this is the only rule that uses it. Also, the function does not appear to have any tests. I wonder if it would be better to move `isES5Constructor` into `consistent-return` instead.